src/Routes/RosSystemDetail/RosSystemDetail.js

```diff
@@ -36,11 +37,16 @@ class RosSystemDetail extends React.Component {
                 cloud_provider: cloudProvider,
                 instance_type: instanceType,
                 idling_time: idlingTime,
-                rating
+                rating,
+                state: state
             } = this.props.rosSystemInfo;
             const { inventoryId } = this.props.match.params;
             return (
                 <Grid className='ros-system-info'>
+                    <GridItem>
+                        <span className='state-value-gap'>State:</span>
+                        <SystemState stateValue={ state }/>
+                    </GridItem>
```

Comment:
@r14chandra after spending sometime going trhough patternfly component my suggestion would be to replace this block by using [`DescriptionList`](https://www.patternfly.org/v4/components/description-list/#compact-horizontal)  and refactor the `Expanded Row` component to use the same. This way we can keep it consistent and the styling will be handled by patternfly. I will also put the styling suggestion which you will need to change.

```suggestion
                  <DescriptionList className='expanded-row' isCompact isHorizontal>
                            <DescriptionListGroup>
                                <DescriptionListTerm>State</DescriptionListTerm>
                                 <DescriptionListDescription>
                                       <SystemState stateValue={ state }/>
                                  </DescriptionListDescription>
                            </DescriptionListGroup>
                        </DescriptionList>
```

for reference go through the conversation [here](https://patternfly.slack.com/archives/C4FM977N0/p1645451871080709) and [sandbox example here](https://codesandbox.io/s/bold-violet-426xfn?file=/index.js:684-704)

feel free to reach out to me for any queries!

Reply:
Thanks @PreetiW for digging deep 🚀 . This looks great to me! I've added the changes as suggested.